client/signup/config/flows.js

```diff
@@ -374,13 +359,24 @@ const Flows = {
 			return flow;
 		}
 
-		if ( user.get() ) {
-			flow = removeUserStepFromFlow( flow );
+		// Replace design-type step with Store NUX flow steps if exists.
+		// Otherwise use Pressable store step for new English users as before
+		if ( includes( flow.steps, 'design-type' ) ) {
+			const enableStore = includes( flow.steps, 'plans' ) || ( flow.meta && flow.meta.enableStore );
+			const storeNuxFlow = enableStore ? Flows.getFlows()[ 'store-nux' ] : undefined;
+			const isNewEnglishUser = ! user.get() && 'en' === i18n.getLocaleSlug();
+			if ( storeNuxFlow || isNewEnglishUser ) {
```

Comment:
Do we even need to check for `isNewEnglishUser` any more? We are filtering people further in the `design-type-with-atomic-store` to allow only for US/CAD to go through the Atomic Store flow and others will be sent to Pressable. And for people who don't even want a store, it will behave as before, even though using the 'store-nux' flow which will be the default one now.

/cc @jhnstn